taxi/views.py

```diff
@@ -79,6 +83,24 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+class CarDriverUpdateView(LoginRequiredMixin, generic.View):
+    @staticmethod
+    def post(
+        request: HttpRequest, pk: int
+    ) -> HttpResponseRedirect | HttpResponseBadRequest:
+        try:
+            car = Car.objects.get(pk=pk)
+        except Car.DoesNotExist:
+            return HttpResponseBadRequest("Car does not exist")
+        drivers = car.drivers.all
```

Comment:
The `drivers` attribute is being accessed as a method with `car.drivers.all`. This should be `car.drivers.all()` to correctly retrieve the queryset of drivers associated with the car.